SQL/New_patches/2019-11-25-Default_value_for_session_submitted.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE session SET Submitted='N' WHERE Submitted IS NULL;
+ALTER TABLE session CHANGE Submitted Submitted ENUM('Y','N') DEFAULT 'N' NOT NULL;
```

Comment:
Since this is going to the 22.0-release branch and #5717 is merged can you update the concatenated patch and put this straight into archived?

Reply:
do we want to merge it though ?

hasnt been tested and not really critical ? might be changing functionality

Reply:
also could you merge https://github.com/aces/Loris/pull/5721

Reply:
not merging it would mean it can't go into a bugfix since it requires an SQL patch, so I'd err on the side of merging it because making it not-nullable with a default makes sense to me.

Reply:
#5721 is already merged